src/controllers/noteController.js

```diff
@@ -0,0 +1,107 @@
+const noteModel = require("../models/noteSchema");
+
+const addNote = async (req, res) => {
+  const { title, note } = req.body;
+  try {
+    //Verified is field is fullfiled
+    if (!title || !note) {
+      return res.status(400).json({
+        success: false,
+        message: "Every field is required",
```

Comment:
Just like in the previous PR, could you make the message more descriptive which field is missing?

Reply:
I thought about it but here we have only two fields 🤭

Reply:
True, but I think it's good to be consistent.